static/app/components/alerts/snoozeAlert.tsx

```diff
@@ -34,30 +40,40 @@ function SnoozeAlert({isSnoozed, onSnooze, projectSlug, ruleId}: Props) {
           },
         }
       );
+
+      setDisabled(false);
+      onSnooze({
+        snooze: !isSnoozed,
+        snoozeCreatedBy: 'You',
+        snoozeForEveryone: target === 'me' ? false : true,
+      });
     } catch (err) {
+      if (err.status === 403) {
+        addErrorMessage(t('You do not have permission to mute this alert'));
+      }
       Sentry.captureException(err);
```

Comment:
Since you are handling this exception, you don't need to create a sentry error here